modules/layers/src/bitmap-layer/bitmap-layer.js

```diff
@@ -180,9 +179,21 @@ export default class BitmapLayer extends Layer {
     );
   }
 
-  draw({uniforms}) {
+  draw(opts) {
+    const {uniforms} = opts;
     const {bitmapTexture, model} = this.state;
-    const {desaturate, transparentColor, tintColor} = this.props;
+    const {image, desaturate, transparentColor, tintColor} = this.props;
+
+    // Update video frame
+    if (
+      bitmapTexture &&
+      image instanceof HTMLVideoElement &&
+      image.readyState > HTMLVideoElement.HAVE_METADATA
+    ) {
+      bitmapTexture.resize({width: image.videoWidth, height: image.videoHeight});
```

Comment:
Looks like `resize` forces `mipmaps` to false? https://github.com/uber/luma.gl/blob/master/modules/webgl/src/classes/texture.js#L198